akka-http-core/src/main/scala/akka/http/scaladsl/model/StatusCode.scala

```diff
@@ -104,6 +105,9 @@ object StatusCodes extends ObjectRegistry[Int, StatusCode] {
   val SwitchingProtocols = reg(i(101)("Switching Protocols", "The server is switching protocols, because the client requested the switch."))
   val Processing         = reg(i(102)("Processing", "The server is processing the request, but no response is available yet."))
 
+  @ApiMayChange
+  val EarlyHints         = reg(i(103)("Early Hints", "The server is likely to send a final response with the header fields included in the response.")) // RFC 8297
```

Comment:
The text is a bit puzzling to me, but that's how the RFC also describes it, so OK :).